sql/src/test/java/io/crate/executor/transport/AlterTableOperationTest.java

```diff
@@ -72,4 +71,35 @@ public void testPrepareRenameTemplateRequests() throws Exception {
 
         assertThat(requests.v2().name(), is(PartitionName.templateName(sourceIdent.schema(), sourceIdent.name())));
     }
+
+    @Test
+    public void testPrepareAlterTableMappingRequest() throws Exception {
+        TableIdent tableIdent = new TableIdent(null, "my_table");
+        Settings settings = Settings.builder()
+            .put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, 1)
+            .put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, 0)
+            .put(IndexMetaData.SETTING_VERSION_CREATED, 0).build();
+
+        MetaData oldMetaData = MetaData.builder()
+            .put(IndexMetaData.builder(PartitionName.templateName(tableIdent.schema(), tableIdent.name()))
+                .putAlias(AliasMetaData.newAliasMetaDataBuilder(tableIdent.name()))
+                .putMapping(Constants.DEFAULT_MAPPING_TYPE, "{\"default\":{\"foo\":\"foo\"}}")
```

Comment:
this is the only thing you need, and pls add a _meta, otherwise it doesn't make real sense. do not create a MetaData, it is not needed at all